data-integration/README.md

```diff
@@ -0,0 +1,341 @@
+# Data integration
+
+## Overview
+
+In this tutorial, you will develop a service that accepts requests to create an employee and a task, update task status, and delete a completed task. Ballerina persistence layer is used to manage data stored in a database. The service will be used to create, retrieve, update, and delete data from the database.
+
+To implement this use case, you will develop a REST service with multiple resources using Visual Studio Code with the Ballerina Swan Lake extension.
+
+Define the data model using the `bal persist` feature and generate the client objects, types, and scripts for the model. Then, define the service and implement the logic to interact with the database.
```

Comment:
Can we rethink this in terms of the flow. Feels disconnected after the preceding sentence.